src/convert.jl

```diff
@@ -28,6 +28,42 @@ function convert(::Type{HPOLYGON1},
     return HPOLYGON1(P.constraints)
 end
 
+"""
+    convert(T::Type{HPOLYGON}, P::VPolygon) where {HPOLYGON<:AbstractHPolygon}
+
+Converts a polygon in vertex representation to a polygon in constraint representation.
+
+### Input
+
+- `HPOLYGON` -- type used for dispatch
+- `P`        -- polygon in vertex representation
+
+### Output
+
+A polygon in constraint representation.
+"""
+function convert(T::Type{HPOLYGON}, P::VPolygon) where {HPOLYGON<:AbstractHPolygon}
```

Comment:
Again, `HPOLYGON` is not needed:
`function convert(T::Type{<:AbstractHPolygon}, P::VPolygon)`

Reply:
Ok, i see what you meant here and in the comment above 👍